<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mText">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nText">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aText">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bText">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>b</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
